parser/remote/README.md

```diff
@@ -0,0 +1,41 @@
+# Build and Run Remote Parsers
+
+Currently, we have two remote parsers, each as a gRPC server in Java:
+
+- hiveql
+- calcite
+
+
+## Development Environment
+
+To make sure that all developers use the same version and configration of development tools like JDK, Maven, and protoc, as well as dependencies like gRPC, we install all of them into a Docker image.
```

Comment:
configration -> configuration

Reply:
Got it. Will fix in the next PR. Thank you for being very detail-oriented!